src/browser/ui/dom/components/LocalEventTrapMixin.js

```diff
@@ -24,10 +24,15 @@ function remove(event) {
 var LocalEventTrapMixin = {
   trapBubbledEvent(topLevelType, handlerBaseName) {
     invariant(this.isMounted(), 'Must be mounted to trap events');
+    var node = this.getDOMNode();
```

Comment:
Add a comment here explaining that this can be null if a React component elsewhere in the tree fatals?